selfdrive/controls/controlsd.py

```diff
@@ -845,7 +845,7 @@ def step(self):
     self.prof.checkpoint("Ratekeeper", ignore=True)
 
     self.is_metric = self.params.get_bool("IsMetric")
-    self.experimental_mode = self.params.get_bool("ExperimentalMode") and self.CP.openpilotLongitudinalControl
+    self.experimental_mode = self.params.get_bool("ExperimentalMode")  # and self.CP.openpilotLongitudinalControl
```

Comment:
Since we want experimental mode to be enabled if no longitudinal, I moved the logic for openpilotLongitudinalControl for end-to-end related features to the subscribers (ui path and planner). Decide if we want to publish something like `endToEndLongitudinal` here as well (need to wait for a CP packet in replay before it shows path

Reply:
I think this is fine

Reply:
oh, one last place is VCruiseHelper for setting vCruise to 65 mph